src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -44,85 +52,207 @@ public static void reportNetworkBalance(Reporter reporter, double activeGenerati
                 .withValue("activeLoad", activeLoad)
                 .withValue("reactiveGeneration", reactiveGeneration)
                 .withValue("reactiveLoad", reactiveLoad)
+                .withSeverity(TypedValue.INFO_SEVERITY)
                 .build());
     }
 
     public static void reportNetworkMustHaveAtLeastOneBusGeneratorVoltageControlEnabled(Reporter reporter) {
         reporter.report(Report.builder()
                 .withKey("networkMustHaveAtLeastOneBusGeneratorVoltageControlEnabled")
                 .withDefaultMessage("Network must have at least one bus with generator voltage control enabled")
+                .withSeverity(TypedValue.ERROR_SEVERITY)
                 .build());
     }
 
-    public static void reportMismatchDistributionFailure(Reporter reporter, int iteration, double remainingMismatch) {
+    public static void reportMismatchDistributionFailure(Reporter reporter, double remainingMismatch) {
         reporter.report(Report.builder()
                 .withKey("mismatchDistributionFailure")
-                .withDefaultMessage("Iteration ${iteration}: failed to distribute slack bus active power mismatch, ${mismatch} MW remains")
-                .withValue(ITERATION, iteration)
+                .withDefaultMessage("Failed to distribute slack bus active power mismatch, ${mismatch} MW remains")
                 .withTypedValue("mismatch", remainingMismatch, OpenLoadFlowReportConstants.MISMATCH_TYPED_VALUE)
                 .withSeverity(TypedValue.ERROR_SEVERITY)
                 .build());
     }
 
-    public static void reportMismatchDistributionSuccess(Reporter reporter, int iteration, double slackBusActivePowerMismatch, int iterationCount) {
+    public static void reportMismatchDistributionSuccess(Reporter reporter, double slackBusActivePowerMismatch, int iterationCount) {
+        String plural = (iterationCount > 1) ? "s" : "";
         reporter.report(Report.builder()
                 .withKey("mismatchDistributionSuccess")
-                .withDefaultMessage("Iteration ${iteration}: slack bus active power (${initialMismatch} MW) distributed in ${iterationCount} iterations")
-                .withValue(ITERATION, iteration)
+                .withDefaultMessage("Slack bus active power (${initialMismatch} MW) distributed in ${iterationCount} inner iteration" + plural)
```

Comment:
don't do that, all reports with the same `key` MUST have the same `defaultMessage`